packages/react-components/src/trace-explorer/trace-explorer-opened-traces-widget.tsx

```diff
@@ -9,6 +9,7 @@ import { faCopy } from '@fortawesome/free-solid-svg-icons';
 import { OpenedTracesUpdatedSignalPayload } from 'traceviewer-base/lib/signals/opened-traces-updated-signal-payload';
 import { ITspClientProvider } from 'traceviewer-base/lib/tsp-client-provider';
 import { WidgetManager } from '@theia/core/lib/browser';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
```

Comment:
Could you please rebase your branch on top of latest master instead of adding a merge commit?

Reply:
> 

Rebased on top of master and fixed merge conflicts. Seems to work since all commits got squashed as well